sass/variables.sass

```diff
@@ -1,5 +1,38 @@
 $yellow: #ffdd57
 
+.spinner 
```

Comment:
why is your spinner styles lives in `variables.sass`?

Reply:
okay, we have created another file main.sass for all future styles and spinner